docs/guides/retry-configuration.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: "azapi"
+page_title: "Feature: Retry Configuration"
+description: |-
+  This guide will describe how to configure the retry behavior in the AzAPI provider. The retry configuration allows you to control how the provider handles failed requests and if it should retry them.
+---
+
+## Why retry?
```

Comment:
Thanks for the doc! Actually we have a doc for the same purpose, would you please combine the two docs?
refs: https://github.com/Azure/terraform-provider-azapi/blob/main/docs/guides/feature_customized_retry.md